dcs/triggers.py

```diff
@@ -8,13 +8,14 @@
 
 
 class TriggerZone:
-    def __init__(self, _id, position: mapping.Point, radius=1500, hidden=False, name=""):
+    def __init__(self, _id, position: mapping.Point, radius=1500, hidden=False, name="", color={1: 1, 2: 1, 3: 1, 4: 0.15}, properties={}):
```

Comment:
You have to be very careful with default dicts/lists/sets/... as this doesn't behave as one would think in python.
In the end every TriggerZone object with a default dict would use the same ref to that dict.
So the usual approach here is to give the default param `None` and check in the `__init__` if it is `None` and create than a default with the intended values.

Reply:
I think I have made the change as you said to.  Let me know if I should have done something different.